integration/live_update_two_images_one_manifest_test.go

```diff
@@ -0,0 +1,92 @@
+//+build integration
+
+package integration
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+const (
+	sparkleUrl = "http://localhost:8100"
```

Comment:
can you move these constants into the function? otherwise we'll run into issues where other functions use them accidentally